src/bind.rs

```diff
@@ -93,6 +93,7 @@ where
 pub enum Protocol {
     Http1 {
         host: Host,
+        is_upgrade: bool,
```

Comment:
Probably warrants a comment.  Specifically, not immediately clear if this means that we _should_ upgrade or if it was already upgraded and we should downgrade... perhaps a more descriptive name would clear this up?

Reply:
Sure, I changed the name to `is_h1_upgrade`, and added comments on it and also where it's used in `bind_service`.

Reply:
looks like there's a reference to the old name in the tests in inbound.rs